shell/common/node_bindings.cc

```diff
@@ -478,7 +475,8 @@ node::Environment* NodeBindings::CreateEnvironment(
                    node::EnvironmentFlags::kHideConsoleWindows |
                    node::EnvironmentFlags::kNoGlobalSearchPaths;
 
-  if (browser_env_ != BrowserEnvironment::kBrowser) {
+  if (browser_env_ != BrowserEnvironment::kBrowser &&
+      browser_env_ != BrowserEnvironment::kUtility) {
```

Comment:
perhaps this should be a positive check for `browser_env_ == kRenderer || browser_env_ == kWorker`?

Reply:
https://github.com/electron/electron/pull/34980/commits/4a3a57985a314f72bf2afa0b8280d812aeec9c6a